src/System.Text.JsonLab.Serialization/System/Text/Json/Serialization/Policies/PropertyValueConverterAttribute.cs

```diff
@@ -9,8 +9,15 @@ public abstract class PropertyValueConverterAttribute : Attribute
     {
         public Type PropertyType { get; protected set; }
 
-        public abstract bool TryGetFromJson(ReadOnlySpan<byte> span, Type type, out object value);
+        // todo: these must to be boxed to object (performance); we need to refactor to add converter interfaces with <T>
 
-        public abstract bool TrySetToJson(object value, out Span<byte> span);
+#if BUILDING_INBOX_LIBRARY
+        public abstract object GetFromJson(ref Utf8JsonReader reader, Type propertyType);
+        public abstract void SetToJson(ref Utf8JsonWriter writer, ReadOnlySpan<byte> name, object value);
+#else
+        //todo: ns20
+        internal abstract object GetFromJson(ref Utf8JsonReader reader, Type propertyType);
```

Comment:
Why are these ifdef'd for ns2.0?

Reply:
I'll change the comment... They have to be internal for now in corefxlab due to a compile error regarding increasing visibility

Reply:
Oh right, good point.